feature/repair/src/main/java/com/iteneum/repair/presentation/RepairView.kt

```diff
@@ -154,8 +190,11 @@ fun RepairView() {
                     )
                     .fillMaxWidth(),
                 textButton = stringResource(id = R.string.text_repair_send_button),
-                onClicked = {/* TODO - Send Repair Button - Add functionality to save repair request info */ }
+                onClicked = {
+                    repairViewModel.onClickSendButton()
+                    navigateToApartment()
```

Comment:
we need to use hight order function here and send the event to the top

Reply:
Modified! Thank you for your help